### PR TITLE
Assistant ID of lower quality

### PR DESCRIPTION
**Pull request by etherware-novice**
# About The Pull Request
(catglove fix vers)
Nanotrasen has decided to start reusing id cards that the clown has destroyed, to save production costs. Assistants got the short end of the stick, being the least respected (and worst) job.

## Why It's Good For The Game
Assistant hate is funny! Also helps make the id card less bland

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

as_idcard

</details>

## Changelog

:cl:
:imageadd: added the 'assistant' card sprite
tweak: makes assistants round-start with the new card
:tweak: adjusts the pda painter and syndicate agent card to use the new sprite
/:cl:

### CODE REVIEW

**Comment by etherware-novice** [comment]
finally finally finally fixed catgloves.dmi

**Comment by nednaZ** [comment]
I can immediately say I dislike the sprite.
It doesn't look like much of anything, it's as if you just drew over the base sprite without any rhyme or reason.

**Comment by NiftyRyanyan** [comment]
If you wanted the “cheap stake” not feel you should have just given them  a hi my name is stickers with a few electronics glued on instead of proper ids

**Comment by etherware-novice** [comment]
> If you wanted the “cheap stake” not feel you should have just given them a hi my name is stickers with a few electronics glued on instead of proper ids

OO good idea

**Comment by etherware-novice** [comment]
as_better

this is perfect

**Comment by NanoCats** [comment]
> this is perfect

It is rotated at an angle while all the other IDs in the game are horizontal

>:(

**Comment by etherware-novice** [comment]
> r

its not supposed to be an *actual* id but just a paper slip with some writing and electronics

File: code/modules/jobs/job_types/assistant.dm [hunk 1]
```diff
@@ -33,6 +33,7 @@ Assistant
 
 /datum/outfit/job/assistant
 	name = "Assistant"
+	id = /obj/item/card/id/job/assistant
```
Comment (Zonespace27) [inline code/modules/jobs/job_types/assistant.dm:36]:
```suggestion
	id = /obj/item/card/id/job/assistant //MONKESTATION ADDITION
```

**Review by Zonespace27 — CHANGES_REQUESTED** [review]
Eh concept overall, not a fan

**Comment by nednaZ** [comment]
Not a fan of either of the sprite options.

**Pull request closed without merging**